CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ HunterGate(
 )
 
 project(evmc)
+set(PROJECT_VERSION "0.1.0.dev0")
```

Comment:
What is `dev0`? Isn't `alpha0` more common?

Reply:
It's "in development" version. `.dev < alpha (a) < beta (b) < rc < ""`.

Reply:
So I'm not sure about versioning syntax, but when I don't I use PEP 440 .